src/views/ForgottenPassword/index.tsx

```diff
@@ -0,0 +1,82 @@
+import AuthPanel from "../../components/AuthPanel";
+import { RouteType } from "../../types";
+import { Typography, Box, Stack, Button, Link, useTheme } from "@mui/material";
+import EastIcon from "@mui/icons-material/East";
+
+import msgs from "./messages";
+import { useIntl } from "react-intl";
+import IconBadge from "../../components/IconBadge";
+import { useState } from "react";
+import TextInput from "../../components/TextInput";
+
+export const ForgottenPasswordView = ({}: RouteType) => {
+  const { formatMessage } = useIntl();
+  const [email, setEmail] = useState("");
+  const theme = useTheme();
+
+  const handleEmailChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setEmail(event.target.value);
+  };
+
+  return (
+    <AuthPanel>
+      <Stack spacing={8}>
+        <Stack spacing={2}>
+          <Stack direction="row" spacing={2} alignItems="center">
+            <IconBadge />
+            <Typography variant="h5" component="h2">
+              {formatMessage(msgs["initialTitle"])}
+            </Typography>
+          </Stack>
+
+          <Typography variant="body1">
+            {formatMessage(msgs["initialDescription"])}
+          </Typography>
+        </Stack>
+
+        <Stack spacing={2}>
+          <TextInput
+            required
+            id="email"
+            label="Email address"
+            name="email"
+            autoComplete="email"
+            autoFocus
+            // error={!!isEmailError}
+            // helperText={isEmailError && formatMessage(msgs[error.code])}
+            value={email}
+            onChange={handleEmailChange}
+          />
+          <Button
+            data-testid="submit-button"
+            type="submit"
+            // disabled={loading || authorizing}
```

Comment:
Will there be a `loading` and `authorized` state?

Reply:
Ok, good point. You've reminded me there are still parts of the Forgotten password flow to do..